docs/ReleaseNotes.md

```diff
@@ -112,6 +112,10 @@ In order to simplify typed record stores, the `FDBRecordStoreBase` class was tur
 
 The `asyncToSync` method of the `OnlineIndexer` has been marked as `INTERNAL`. Users should transition to using one of the `asyncToSync` methods defined on either `FDBDatabase`, `FDBRecordContext`, or `FDBDatabaseRunner`. This method may be removed from our public API in a later release (see [Issue # 473](https://github.com/FoundationDB/fdb-record-layer/issues/473)).
 
+### 2.5.54.17
```

Comment:
This needs a release note in the `next_release` saying that it incorporates everything in the `2.5.54.17` release.

Reply:
Done.